File: GTTS/INL.VettingService.Models/IPersonVettingStatus_Item.cs
using System;

namespace INL.VettingService.Models
{
    public interface IPersonVettingStatus_Item
    {
        long PersonID { get; set; }
        long? TrainingEventID { get; set; }
        int VettingBatchStatusID { get; set; }
        string BatchStatus { get; set; }
        int VettingPersonStatusID { get; set; }
        string PersonsVettingStatus { get; set; }
        DateTime? DateLeahyFileGenerated { get; set; }
        DateTime? VettingBatchStatusDate { get; set; }
        DateTime? ExpirationDate { get; set; }
        bool RemovedFromVetting { get; set; }
        DateTime? VettingStatusDate { get; set; }
        int VettingBatchTypeID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/iperson-vetting-status_item.ts




export interface IPersonVettingStatus_Item {
  
	PersonID: number;
	TrainingEventID?: number;
	VettingBatchStatusID: number;
	BatchStatus: string;
	VettingPersonStatusID: number;
	PersonsVettingStatus: string;
	DateLeahyFileGenerated?: Date;
	VettingBatchStatusDate?: Date;
	ExpirationDate?: Date;
	RemovedFromVetting: boolean;
	VettingStatusDate?: Date;
	VettingBatchTypeID: number;

}

